### DISCUSSION TITLE
Publishing layer from postgresql directly in geoServer

### DISCUSSION THREAD

**Discussion by rfazimut** [GeoCat/qgis-bridge-plugin#190] [Q&A]
We are trying to publish from GeoCat 4.5.1 to GeoServer 2.26.  The layer refers to a postgresql table. We have been unsuccessful on publishing that layer so that it refers to the postgresql database.  We a previous version of Geocat, we had an issue with with invalid media type.  With the current version, I cannot add a database or iterate the list of database defined on the server with this message 
 datastores = list(chain.from_iterable(worker_result))
TypeError: 'KeyError' object is not iterable

**Comment by GeoSander** [comment]
Related to #191.

Sounds like a regression to me. May well be that if I fix this, the old "invalid media type" error will be back...
